### DISCUSSION TITLE
arm架构安装问题

### DISCUSSION THREAD

**Discussion by cohhi** [natfrp/wiki#301] [使用问题] [answered]
系统`Centos 7.9`
架构`aarch64`

- 下载
![02](https://user-images.githubusercontent.com/71576964/229434727-8b272151-9065-4265-9c50-2e41cbf62c76.PNG)

- md5sum也没有问题,但是`frpc -v`命令无效
![04](https://user-images.githubusercontent.com/71576964/229434825-679f4e61-9dde-43bb-bf74-d7ca86882927.PNG)

**Comment by fakeboboliu** [comment, marked as answer]
您好，
`./frpc -v`
